.github/workflows/release-automation-maven.yml

```
@@ -0,0 +1,183 @@
+name: Release Automation Maven
```

Admittedly the above is clearer, but when viewing the actions page it will show up as a separate workflow without any history. To show up as the old workflow with its history intact we should name the file as `auto-release.yml` and keep the old name.
```suggestion
name: Automated Release
```

---

sure will revert those changes as well as adding the chart version input back